### DISCUSSION TITLE
Autoplay Latest Message does not work for tts

### DISCUSSION THREAD

**Discussion by FredNov** [danny-avila/LibreChat#4535] [Troubleshooting]
### What happened?

 Autoplay Latest Message does not work for tts

### Steps to Reproduce

Enable  Autoplay Latest Message on Firefox or Chrome or edge

### What browsers are you seeing the problem on?

Firefox, Chrome, Microsoft Edge

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
It works. It could be a configuration issue, which there is not enough information here to determine.

**Comment by danny-avila** [comment]
To add more information here, on desktop, the response message needs to be long enough for the audio element to first unmute and then play, which unfortunately is a browser limitation. On firefox there are also some issues with autoplay but not chrome. The audio device itself factors into this.

The autoplay issue is more nuanced for mobile, since there are strict browser policies for handling audio on these devices:
https://developer.chrome.com/blog/autoplay/

Autoplay is nice to have but it requires a lot of investment into development time to work out all these quirks, across multiple devices, and is not seen as a priority feature to focus on.

**Reply by FredNov** [reply]
yeah it could be nice to have it but I get it. 
I would personally use it for everything if LibreChat had autoplay and properly working STT, because the microphone icon has never worked for me on any browser or device (I suspect it cannot properly request microphone permissions).
There are literally no good options. Gemini app on android lacks granular controls. Then openai which doesn't work through vpn. 
But anyway thanks a lot for your work and what we have.

**Reply by danny-avila** [reply]
We will start focusing on it more towards the end of the year and start of the next. This is just my realistic timeline. Thanks for trying out LibreChat
